src/components/composite/Header.stories.js

```diff
@@ -29,11 +35,9 @@ const props = {
   },
 };
 
-storiesOf('Composite|Header', module)
-  .addParameters({ component: Header })
-  .add('default', () => <Header {...props} />)
-  .add('inverted', () => (
-    <BlueBackground>
-      <Header isInverted {...props} />
-    </BlueBackground>
-  ));
+export const Inverted = Story.bind({});
+Inverted.args = {
+  ...Default.args,
+  isInverted: true,
+};
+Inverted.decorators = [story => <BlueBackground>{story()}</BlueBackground>];
```

Comment:
Yeah I am a big fan of this 👍 (the decorator).

Reply:
You and me. This way of organizing the stories and bringing in items like decorators closer to the story feels more organic to me.